include/nbl/builtin/hlsl/bxdf/reflection.hlsl

```diff
@@ -1,40 +1,22 @@
-// Copyright (C) 2018-2020 - DevSH Graphics Programming Sp. z O.O.
+// Copyright (C) 2018-2025 - DevSH Graphics Programming Sp. z O.O.
 // This file is part of the "Nabla Engine".
 // For conditions of distribution and use, see copyright notice in nabla.h
 #ifndef _NBL_BUILTIN_HLSL_BXDF_REFLECTION_INCLUDED_
 #define _NBL_BUILTIN_HLSL_BXDF_REFLECTION_INCLUDED_
 
-#include <nbl/builtin/hlsl/bxdf/common.hlsl>
+#include "nbl/builtin/hlsl/bxdf/reflection/lambertian.hlsl"
+#include "nbl/builtin/hlsl/bxdf/reflection/oren_nayar.hlsl"
+#include "nbl/builtin/hlsl/bxdf/reflection/beckmann.hlsl"
+#include "nbl/builtin/hlsl/bxdf/reflection/ggx.hlsl"
```

Comment:
we need a basic "DeltaDistribution" reflection BxDF (generate always gives you the reflection vector around `N`, pdf is infinite and quotient is 1.0)